encoding.bs

```diff
@@ -18,6 +18,7 @@ Markup Shorthands: css off
 Translate IDs: dictdef-textdecoderoptions textdecoderoptions,dictdef-textdecodeoptions textdecodeoptions
 </pre>
 
+<style>@import 'visualization-colors.css';</style>
```

Comment:
According to Tab we can use `<link>` and Bikeshed will move it correctly in the output.